.clang-format

```diff
@@ -13,53 +14,95 @@ AllowShortFunctionsOnASingleLine: Empty
 AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
 AlwaysBreakAfterDefinitionReturnType: None
+AlwaysBreakAfterReturnType: None
 AlwaysBreakBeforeMultilineStrings: true
-AlwaysBreakTemplateDeclarations: true
+AlwaysBreakTemplateDeclarations: Yes
```

Comment:
Should this be `Yes`? All other boolean values are `true` or `false`.

Reply:
`Yes` in this context is `BTDS_Yes`, the type of this is not `bool` but `BreakTemplateDeclarationsStyle`. There is `BTDS_No`, but there is also `BTDS_Multiline`, so it actually isn't just a boolean.
Source: https://clang.llvm.org/docs/ClangFormatStyleOptions.html